doc/tutorials/java-kdb.md

```diff
@@ -136,6 +136,20 @@ try (KDB kdb = KDB.open()) {
 }
 ```
 
+Another way to traverse is to use the Stream API which was introduced with Java 8:
+
+```java
+var keyNamespace = Key.create("user:/");           // select a namespace from which all keys should be fetched
+try (KDB kdb = KDB.open()) {
+    var keySet = kdb.get(keyNamespace);            // fetch all keys into a new key set
+    keySet.forEach(key -> System.out.printf("%s: %s%n", key.getName(), key.getString()));                              // directly format-print all key value pairs using foreach
+    // or
+    keySet.stream().map(key -> String.format("%s: %s", key.getName(), key.getString())).forEach(System.out::println);  // map the key value paris to the desired format first and then print them using foreach and a function reference
+} catch (KDBException e) {
+    e.printStackTrace();
+}
+```
+
```

Comment:
Thank you for you suggestion. This is now mentioned on the bottom. However, the CI currently fails because it detects broken links (in a documented untouched by this PR) which are reachable from my home network. Re-running it multiple times did not resolve this.

Reply:
Please ignore the failing check for https://researchcourse.pbworks.com/f/structural%2Bengineering.pdf for now. If it doesn't resolve itself within the next few days, well add it to `tests/linkchecker.whitelist`